isso/ext/notifications.py

```diff
@@ -13,20 +13,27 @@
 from email.header import Header
 from email.mime.text import MIMEText
 
+from pathlib import Path
+from string import Template
 from urllib.parse import quote
 
 import logging
-logger = logging.getLogger("isso")
 
 try:
     import uwsgi
 except ImportError:
     uwsgi = None
 
-from isso import local
+from isso import dist, local
+from isso.views.comments import isurl
 
 from _thread import start_new_thread
 
+from requests import HTTPError, Session
```

Comment:
Please don't introduce another dependency needlessly, but I think I already commented on that.